platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -308,6 +308,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)mapView:(MGLMapView *)mapView didChangeUserTrackingMode:(MGLUserTrackingMode)mode animated:(BOOL)animated;
 
+/**
+ Returns a CGPoint value to offset the user location annotation, relative to the
```

Comment:
`CGPoint` is already present in the declaration, so I’d maybe describe it here in lay terms. It’s an absolute screen coordinate, not an offset (which would be `CGVector`):

> Returns a screen coordinate at which to position the user location annotation…